source/host_resolver.c

```diff
@@ -1474,7 +1474,6 @@ static int default_resolve_host(
             res(resolver, host_name, AWS_OP_SUCCESS, &callback_address_list, user_data);
         } else {
             res(resolver, host_name, aws_last_error(), NULL, user_data);
```

Comment:
I have no idea what value `aws_last_error()` will return here
bug?

Reply:
I have updated it again, now that I understand more about C error handling.